src/java/org/pantsbuild/tools/jar/JarEntyCopier.java

```diff
@@ -0,0 +1,173 @@
+// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package org.pantsbuild.tools.jar;
+
```

Comment:
This filename is misspelled... maybe it always was?

Reply:
Good catch, it actually was correctly named in the original branch, just fixed it.